### PR TITLE
Added license and basic files/folder

### PR DESCRIPTION
**Pull request by mansikulkarni96**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Click==7.0
```
Comment (samvarankashyap) [inline requirements.txt]: It is not advisable to hardcode the requirements for application. 
Please make them base requirements. 
For example:
```
click==7.0
```
becomes
```
click>=7.0
```

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# IDE files
+.idea/
+#virtualenv
+venv/
```
Comment (samvarankashyap) [inline .gitignore:4]: Make sure you add other files to gitignore like 
```
*.pyc
*.swp
.cache
.pytest_cache
```

**Review by samvarankashyap — CHANGES_REQUESTED** [review]
LGTM apart from the changes requested.

**Comment by samvarankashyap** [comment]
@mansikulkarni96 Please make sure you give a quick description of what this PR does.
and what issues it fixes. for example this current PR fixes #1 and #2

**Comment by samvarankashyap** [comment]
Adding to @14rcole to the reviewers' list

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+Flask==1.0.3
+itsdangerous==1.1.0
+Jinja2==2.10.1
+MarkupSafe==1.1.1
+Werkzeug==0.15.4
```
Comment (14rcole) [inline requirements.txt]: How many of these are directly required by the application and how many are flask dependencies?  I may be wrong, but if they're just flask dependencies then they should be installed automatically.  @samvarankashyap you may know more about this than I do

Comment (samvarankashyap) [reply requirements.txt]: @mansikulkarni96 
I agree with @14rcole 
according to flask project: 
https://github.com/pallets/flask/blob/master/setup.py
```
        "Werkzeug>=0.15",
        "Jinja2>=2.10.1",
        "itsdangerous>=0.24",
        "click>=5.1",
```
are already included as part of flask installation. 
Thus making them unnecessary for the current project if the flask is included. 
You can remove them from the current requirements file. 
@14rcole Thanks for pointing it out.

Comment (mansikulkarni96) [reply requirements.txt]: I will remove them all, thanks @14rcole  and @samvarankashyap

**Review by samvarankashyap — APPROVED** [review]
LGTM

**Pull request merged**